src/functions.jl

```diff
@@ -569,16 +614,11 @@ for T in (:Taylor1, :TaylorN)
                 return nothing
             end
```

Comment:
Same comment about order of `if` and `for`

Reply:
Done 👍